wp-content/themes/oceanwp-child/assets/css/template-styles.css

```diff
@@ -273,4 +276,10 @@ hr{
 .ini-supporters{
```

Comment:
can you please change this to `initiative-supporters`?

Reply:
Done